### DISCUSSION TITLE
2tand NZP Reboot Map

### DISCUSSION THREAD

**Discussion by MotoLegacy** [nzp-team/nzportable#324] [Map Releases]
## This is a manual migration of forum user BCDeshiG's map release `2tand`. Below contains the original post content. You can view the original post [here](https://nzportable.forumotion.com/t2237-2tand-nzp-reboot-map).

Another challenge map akin to the warehouse starting room.
There are 4 perks in this map along with mystery box and pap.
Should also be ready for custom weapon support when it fully arrives.
This is a challenge map made for NZP Reboot.

Please inform me of any problems with the map; especially waypoints.

__Screenshots:__
![0](https://github.com/nzp-team/nzportable/assets/26030250/66f3c6d7-83c4-4077-a132-1a61cf0956ae)
![1](https://github.com/nzp-team/nzportable/assets/26030250/a14f89c0-9d98-4ec1-aa2c-916d498db725)

# [DOWNLOAD](https://github.com/nzp-team/nzportable/files/12147956/2tand.zip)

**Comment by BCDeshiG** [comment]
Hardly worth playing tbh.

**Reply by MotoLegacy** [reply]
Anything that still had an active download link got archived 🤷 I don't judge quality.
